File: 백준/Bronze/19532. 수학은 비대면강의입니다/수학은 비대면강의입니다.js
let fs = require("fs");
let [a,b,c,d,e,f] = fs.readFileSync("/dev/stdin").toString().trim().split(" ").map(Number);

for(let x = -999; x<= 999; x++){
  for(let y = -999; y<=999; y++){
    if(a*x + b*y === c && d*x + e*y === f){
      console.log(x , y);
      break;
    }
  }
}

File: 백준/Bronze/19532. 수학은 비대면강의입니다/수학은 비대면강의입니다.py
import sys
input = sys.stdin.readline

A, B , C , D , E ,F = map(int , input().split())

for x in range(-999 , 1000):
    for y in range(-999 , 1000):
        if A*x + B*y == C:
            if D*x + E*y == F:
                print(x , y)
                break


